src/components/PostPageDetail/index.js

```diff
@@ -34,6 +42,7 @@ import LoadingWithoutModal from '../LoadingWithoutModal';
 import {withInteractionsManaged} from '../WithInteractionManaged';
 import Content from './elements/Content';
 import usePostDetail from './hooks/usePostDetail';
+import {Shimmer} from '../Shimmer/Shimmer';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Shimmer` component has been imported presumably to provide a loading effect while fetching data. Make sure that the `Shimmer` component is properly implemented and tested.
```diff
+ import {Shimmer} from '../Shimmer/Shimmer';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->